### DISCUSSION TITLE
Discussion | Tuesday, 19 November 2024, 8:00 PM

### DISCUSSION THREAD

**Discussion by sharafdin** [dugsiiyeinc/dhaqsoPay#41] [Zoom Links]
Caawa aan isku imaano Insha'Allah!

**Zoom Link::**
https://us06web.zoom.us/j/88674441882?pwd=wdpCa0qQLpaaacwKVsKO6fqK1tjY4O.1

### Mentions
@dugsiiyeinc/dhaqsopay-extension-developers 
@dugsiiyeinc/dhaqsopay-server-developers 
@dugsiiyeinc/dhaqsopay-website-developers

**Comment by abdinasir-Tman** [comment]
inshaa allah

**Comment by shiinedev** [comment]
Ok insha`Allah

**Comment by ducesane** [comment]
insha allah

**Comment by sharafdin** [comment]
waan joognaa dhalinyaro
